core-bundle/src/EventListener/DataContainer/PageTypeOptionsListener.php

```diff
@@ -51,6 +53,10 @@ public function __invoke(DataContainer $dc): array
     {
         $options = array_unique(array_merge(array_keys($GLOBALS['TL_PTY']), $this->pageRegistry->keys()));
 
+        if ('tl_user' === $dc->table || 'tl_user_group' === $dc->table) {
```

Comment:
Should this not be added after the event has been dispatched?

Reply:
No. The event is meant to limit the page options (e.g. Only show the root option on pid=0). That does not apply here.